app/views/recipes/index.html.erb

```diff
@@ -0,0 +1,14 @@
+<h2>RECIPE LIST GOES HERE</h2>
+<section>
+  <h1>Recipes</h1>
+
+  <div>
+    <% if @recipes.empty? %>
+      <h2 class="text-center">The list is empty.</h2>
+    <% else %>
+      <% @recipes.each do |recipe| %>
+      <%= render('card', recipe:)  %>
+      <% end %>
+    <% end %>
```

Comment:
- [ ] We should be able to delete a recipe maybe you can add a button to delete a recipe from the UI and the DB using the destroy method in your recipe_controller.